src/ProjectTemplates/BlazorTemplates.Tests/playwrightSettings.json

```diff
@@ -4,18 +4,12 @@
   "BaseArtifactsFolder": ".",
   "GlobalBrowserOptions": {
     "ChromiumSandbox": true,
-    "DumpIO": true,
-    "IgnoreHTTPSErrors": true,
     "Headless": true,
     "Timeout": 30000
   },
   "GlobalContextOptions": {
-    "RecordVideo": {
-      "Dir": "videos"
-    },
-    "RecordHar": {
-      "Path": "har"
-    },
+    "RecordVideoDir": "videos",
+    "RecordHarPath": "har",
```

Comment:
I can't make a comment in line 25 (github limitation) but you can remove the args with --ignore-certiicate-errors since you already do it in line 19.